tests/config.bats

```diff
@@ -188,100 +138,40 @@ load helpers
-
-  buildah --debug=false inspect --type=image --format '{{.Docker.Config.WorkingDir}}' scratch-image-docker | grep /tmp
-  buildah --debug=false inspect --type=image --format '{{.OCIv1.Config.WorkingDir}}' scratch-image-docker | grep /tmp
-  buildah --debug=false inspect --type=image --format '{{.Docker.Config.WorkingDir}}' scratch-image-oci | grep /tmp
-  buildah --debug=false inspect --type=image --format '{{.OCIv1.Config.WorkingDir}}' scratch-image-oci | grep /tmp
-
-  buildah --debug=false inspect --type=image --format '{{.Docker.Config.Labels}}' scratch-image-docker | grep LABEL:VALUE
-  buildah --debug=false inspect --type=image --format '{{.OCIv1.Config.Labels}}' scratch-image-docker | grep LABEL:VALUE
-  buildah --debug=false inspect --type=image --format '{{.Docker.Config.Labels}}' scratch-image-oci | grep LABEL:VALUE
-  buildah --debug=false inspect --type=image --format '{{.OCIv1.Config.Labels}}' scratch-image-oci | grep LABEL:VALUE
-
-  buildah --debug=false inspect --type=image --format '{{.Docker.Config.Labels}}' scratch-image-docker | grep 'exec:podman run -it --mount=type=bind,bind-propagation=Z,source=foo,destination=bar /script buz'
-  buildah --debug=false inspect --type=image --format '{{.OCIv1.Config.Labels}}' scratch-image-docker | grep 'exec:podman run -it --mount=type=bind,bind-propagation=Z,source=foo,destination=bar /script buz'
-  buildah --debug=false inspect --type=image --format '{{.Docker.Config.Labels}}' scratch-image-oci | grep 'exec:podman run -it --mount=type=bind,bind-propagation=Z,source=foo,destination=bar /script buz'
-  buildah --debug=false inspect --type=image --format '{{.OCIv1.Config.Labels}}' scratch-image-oci | grep 'exec:podman run -it --mount=type=bind,bind-propagation=Z,source=foo,destination=bar /script buz'
-
-  buildah --debug=false inspect --type=image --format '{{.Docker.Config.StopSignal}}' scratch-image-docker | grep SIGINT
-  buildah --debug=false inspect --type=image --format '{{.OCIv1.Config.StopSignal}}' scratch-image-docker | grep SIGINT
-  buildah --debug=false inspect --type=image --format '{{.Docker.Config.StopSignal}}' scratch-image-oci | grep SIGINT
-  buildah --debug=false inspect --type=image --format '{{.OCIv1.Config.StopSignal}}' scratch-image-oci | grep SIGINT
+  check_matrix 'Config.User'         'likes:things'
+  check_matrix 'Config.ExposedPorts' 'map[12345:{}]'
+  check_matrix 'Config.Env'          '[VARIABLE=VALUE1,VALUE2]'
+  check_matrix 'Config.Entrypoint'   '[/bin/sh -c /ENTRYPOINT]'
+  check_matrix 'Config.Cmd'          '[COMMAND-OR-ARGS]'
+  check_matrix 'Config.Volumes'      'map[/VOLUME:{}]'
+  check_matrix 'Config.WorkingDir'   '/tmp'
+  check_matrix 'Config.Labels'       '.*LABEL:VALUE'
+  check_matrix 'Config.Labels'       '.*exec:podman run -it --mount=type=bind,bind-propagation=Z,source=foo,destination=bar /script buz'
+  check_matrix 'Config.StopSignal'   'SIGINT'
```

Comment:
nit of a nit, might be nice to order the checks alphabetically here by the checked value.  ie. Config.User, Config.Env, etc.

Reply:
Maybe in a future PR? I wanted to keep those diffs (at least partly) readable

Reply:
yes, I'm fine with doing that or any further changes in a second PR.  If this goes green, I'd like to merge it.